exonum-java-binding/core/rust/src/runtime/library_loader.rs

```diff
@@ -23,8 +23,8 @@ pub extern "system" fn Java_com_exonum_binding_util_LibraryLoader_nativeGetLibra
     env.new_string(get_lib_version()).unwrap().into_inner()
 }
 
-// Returns the exact value of the `version` field from the library's Cargo.toml configuration file.
-fn get_lib_version() -> &'static str {
+/// Returns the exact value of the `version` field from the library's Cargo.toml configuration file.
+pub fn get_lib_version() -> &'static str {
```

Comment:
Shall it be moved to `utils` for exonum-java to depend on it :thinking: ?

Reply:
I was also thinking about this but didn't end up with a name for a module except "misc", "common", "other" etc 🙃